cli/internal/auth/logout.go

```diff
@@ -1,22 +1,22 @@
 package auth
 
 import (
+	"errors"
 	"fmt"
 
 	cqapiauth "github.com/cloudquery/cloudquery-api-go/auth"
 	"github.com/cloudquery/cloudquery-api-go/config"
 )
 
 func Logout() error {
-	err := cqapiauth.RemoveRefreshToken()
-	if err != nil {
-		return fmt.Errorf("failed to remove refresh token: %w", err)
+	var errs []error
+	if err := cqapiauth.RemoveRefreshToken(); err != nil {
+		errs = append(errs, fmt.Errorf("failed to remove refresh token: %w", err))
 	}
 
-	err = config.UnsetValue("team")
-	if err != nil {
-		return fmt.Errorf("failed to unset team: %w", err)
+	if err := config.UnsetValue("team"); err != nil {
```

Comment:
@candiduslynx were you able to reproduce the issue? Seems like it shouldn't happen normally since we're already clearing the team from the config

Reply:
If we have an issue removing refresh token this might happen

Reply:
I understand, but did you ran into an issue where we get that error to remove the token? Based on the issue description not sure it's related as no errors where reported during logout